logfire/_internal/scrubbing.py

```
@@ -321,20 +335,35 @@ def scrub(self, path: JsonPath, value: Any) -> Any:
                 else:
                     return json.dumps(self.scrub(path, value))
         elif isinstance(value, Sequence):
-            return [self.scrub(path + (i,), x) for i, x in enumerate(cast('Sequence[Any]', value))]
+            sequence = cast('Sequence[Any]', value)
+            sequence_result: list[Any] | None = None
+            for i, old_value in enumerate(sequence):
+                new_value = self.scrub(path + (i,), old_value)
```

Ruff (0.16.1)

[warning] Consider `(*path, i)` instead of concatenation

Replace with `(*path, i)`

(RUF005)